### PR TITLE
feat: chat image

### PR DESCRIPTION
**Pull request by paul-nguyen-goldenowl**
## Ticket
chat image

## What does PR do?

- [x]  send and display images

## Video 
send_message

### CODE REVIEW

File: simplechat/src/main/java/com/ln/simplechat/ui/chat/ChatFragment.kt [hunk 1]
```diff
@@ -93,12 +121,21 @@ class ChatFragment : Fragment(R.layout.chat_fragment) {
...
         fun newInstance(channel: Channel) =
             ChatFragment().apply {
                 arguments = Bundle().apply {
                     putParcelable(CHANNEL, channel)
                 }
             }
     }
+
+    override fun openPreview(position: Int, data: List<ChatMedia>) {
+//        activity?.supportFragmentManager?.commit {
```
Comment (paul-nguyen-goldenowl) [inline simplechat/src/main/java/com/ln/simplechat/ui/chat/ChatFragment.kt:136]: todo: preview image

File: simplechat/src/main/java/com/ln/simplechat/ui/chat/ChatFragment.kt [hunk 2]
```diff
@@ -93,12 +121,21 @@ class ChatFragment : Fragment(R.layout.chat_fragment) {
...
         const val CHANNEL = "CHANNEL"
+
         fun newInstance(channel: Channel) =
             ChatFragment().apply {
                 arguments = Bundle().apply {
                     putParcelable(CHANNEL, channel)
                 }
             }
     }
+
```
Comment (paul-nguyen-goldenowl) [inline simplechat/src/main/java/com/ln/simplechat/ui/chat/ChatFragment.kt:134]: - [ ] Preview Image

File: simplechat/src/main/java/com/ln/simplechat/ui/chat/ChatAdapter.kt [hunk 3]
```diff
@@ -58,25 +71,29 @@ class ChatAdapter(
...
         var messengerAvatar: ImageView = itemView.findViewById(R.id.messenger_avatar)
-        var messageImage: ImageView = itemView.findViewById(R.id.message_image)
+        var messageImageRcv: RecyclerView = itemView.findViewById(R.id.rcv_message_image)
         var messageVideo: VideoView = itemView.findViewById(R.id.message_video)
         var timestamp: TextView = itemView.findViewById(R.id.tv_timestamp)
 
         fun bind(item: Message) {
+            val outGoing = item.sender == currentUserId
             toggleVisibility(item)
             if (item.text != null) {
```
Comment (dustin-goldenowl) [inline simplechat/src/main/java/com/ln/simplechat/ui/chat/ChatAdapter.kt]: use "when" for this case

File: simplechat/src/main/java/com/ln/simplechat/ui/chat/ChatAdapter.kt [hunk 4]
```diff
@@ -86,15 +103,15 @@ class ChatAdapter(
         private fun toggleVisibility(item: Message) {
             if (item.text != null) {
```
Comment (dustin-goldenowl) [inline simplechat/src/main/java/com/ln/simplechat/ui/chat/ChatAdapter.kt]: use "when" for this case

File: simplechat/src/main/java/com/ln/simplechat/ui/chat/ChatFragment.kt (hunk 1 above)
Comment (dustin-goldenowl) [inline simplechat/src/main/java/com/ln/simplechat/ui/chat/ChatFragment.kt]: remove if unuse

**Pull request merged**